src/reducers/timelines.js

```diff
@@ -1,49 +1,35 @@
 import { ADD_TOOTS } from '../constants/toots'
 
-const initialState = {
-	home : {
-		// isLoading ?
-		items : []
-	},
-	local : {
-		// isLoading ?
-		items : []
-	},
-	federated : {
-		// isLoading ?
-		items : []
-	}
-}
-
-function normalizeToots(timeline, state, toots) {
-	const currentToots = state[timeline].items
+const initialState = {}
 
-	return {
-		items : toots.map(toot => toot.id).concat(currentToots)
-	}
+function normalizeToots(newtoots, oldToots) {
+	// TODO / FIXME this is ugly.
+	return [...new Set(newtoots.map(toot => toot.id).concat(oldToots))]
```

Comment:
Let's use simple objects for this :)
I never know when to use Set > Object but in this very case it makes it hard to see the state of our store with the redux-tools, and doesn't bring much to the table anyway.